src/internal/interpreter/profile-parameter-validator.ts

```diff
@@ -483,12 +483,12 @@ export class ProfileParameterValidator implements ProfileVisitor {
     if (!this.namedDefinitionsInitialized) {
       node.definitions
         .filter(
-          (definition): definition is NamedFieldDefinitionNode =>
-            definition.kind === 'NamedFieldDefinition'
+          (definition): definition is NamedModelDefinitionNode =>
+            definition.kind === 'NamedModelDefinition'
         )
```

Comment:
Why not use profile ast utils? `isNamedModelDefinitionNode` and `isNamedFieldDefinitionNode` should work.